packages/camera/camera_android/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_android
 description: Android implementation of the camera plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/camera/camera_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.10.3
+version: 0.10.4
```

Comment:
Do we have a presubmit that makes sure we update this? I think it is something I would forget.

Reply:
Yes, the repo_checks test checks for that! Just FYI, there are also labels to override it when not needed!